tests/cli_test.py

```diff
@@ -90,7 +97,7 @@ def test_simple(runner):
 
     result = runner.invoke(main_khal)
     assert not result.exception
-    assert result.output == 'No events\n'
+    assert strip_ansi_escapes(result.output) == 'No events\n'
```

Comment:
It seems to remove colour codes, but not bold codes (and it's bold ANSI escapes we're getting in this case).